### ISSUE TITLE
Shell and Shell Panel - Ensure implicit list markup is avoided - (2036570580)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7827]
### Violation: 
 Ensure implicit list markup is avoided 
![image](https://github.com/Esri/calcite-design-system/assets/5023024/c14f9e65-db71-4c79-a063-c75754187ff4)

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) 

### Severity: 
 6 

### Media Type: 
 Lists 

--- 
### Areas for Remediation: 
- Title: Shell and Shell Panel - Ensure implicit list markup is avoided - (2036570580) 
- Module: Shell and Shell Panel 
**Pattern:** List- Missing list markup.]

**Issue**
There is content that appears as a list but does not use list markup. Examples include:
- Add action, for display purposes only
- Layers action, for display purposes only, etc content in the left and right side panel.
- A filter-enabled value list for the item.

**Occurrences**
29 Shell and Shell Panel
30 Shell Center Row

**User Impact**
Screen reader users will have difficulty understanding that this content is a list.

**Code Reference** 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Add action, for display purposes only" class="button button--text-visible"><div aria-hidden="true" class="icon-container"><calcite-icon aria-hidden="true" icon="plus" scale="s" calcite-hydrated=""></calcite-icon><div class="slot-container"><slot></slot></div></div><div class="text-container text-container--visible">Add action</div></button>
(...)
<button aria-busy="false" aria-disabled="false" aria-label="Collapse" class="button button--text-visible">...
</button>
 ``` 
**Suggestion**
Ensure implicit list markup is avoided. Lists must use ``` <ul> ```, ``` <ol> ```, ``` <li> ```, ``` <dl> ```, ``` <dt> ```, and ``` <dd> ``` elements as appropriate.

**Compliant Code Example** 
 ```
<ul>
<li><button aria-busy="false" aria-disabled="false" aria-label="Add action, for display purposes only" class="button button--text-visible"><div aria-hidden="true" class="icon-container"><calcite-icon aria-hidden="true" icon="plus" scale="s" calcite-hydrated=""></calcite-icon><div class="slot-container"><slot></slot></div></div><div class="text-container text-container--visible">Add action</div></button></li>
(...)
<li><button aria-busy="false" aria-disabled="false" aria-label="Collapse" class="button button--text-visible">...
</button></li>
</ul>
 ```
 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by macandcheese** [comment]
This seems unrelated to Shell or Shell Panel - referenced code is not part of these components. Could we close this one?

**Comment by geospatialem** [comment]
Closing, looks like a false positive due to testing.
